synth/src/version.rs

```diff
@@ -0,0 +1,95 @@
+use anyhow::Result;
+use core::option::Option;
+use core::option::Option::{None, Some};
+use core::result::Result::Ok;
+use core::time::Duration;
+use semver::Version;
+use serde_json::map::Map;
+use serde_json::value::Value;
+use reqwest::header::USER_AGENT;
+use crate::cli::config;
+
+
+pub fn version() -> String {
+    env!("CARGO_PKG_VERSION").to_string()
+}
+
+fn version_semver() -> Result<Version> {
```

Comment:
This can just panic if parsing fails. The version is controlled by us, so there's no need to do fancy error handling.